<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open_com">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close_com">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_com_num">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_band">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>47</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>11</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>21</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>访问IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_IP">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>访问间隔：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>68</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>通讯中断判定时长：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>65</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_interval_time">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>45</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_disconnect_time">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_switch" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>94</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_net_set_status">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>93</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_reveive_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>541</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_tx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>功能键</string>
    </property>
    <widget class="QPushButton" name="pushButton_YV1_close">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>YV1吸合</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_YV1_open">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>YV1释放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_YV2_close">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>YV2吸合</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_YV2_open">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>YV2释放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_testConnect">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>联网测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
